bin/cron/dms_tasks

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
+
+# This script runs the AWS Data Migration Service tasks that replicate data from
+# our reporting RDS database to our Redshift database. It is highly tied to the
+# AWS DMS tasks (https://console.aws.amazon.com/dms/home?region=us-east-1#tasks:)
+# and the CDO.dms_tasks secret.
+
+require 'cdo/hip_chat'
+
+# The list of DMS tasks to start, as well as the length of time that should
+# elapse before starting the next task (an overestimate of completion time).
+# The hash keys are expected to be keys in CDO.dms_tasks through which the task
+# ARNs are retrieved.
+TASKS = {
+  oneoff_survey_hierarchy: {
```

Comment:
Why are all the tasks named `oneoff`? The name implies that they shouldn't belong in a long-lived scheduled task such as this.